README.md

```diff
@@ -282,6 +284,31 @@ use:
 default_mimetype=text/plain
 ```
 
+#### File type recognition
+
+It is possible to specify the recognizer of FILE type.
+For example, if your server of static pages set defaults recognizer
+as your_file_recognizer function, use:
+
+```yaml
+filetype_recognizer=module:your_file_recognizer
+```
+
+Value is recognizer, which can be defined as:
+* strings in the form `"module:function"`, which name the finder
+  function to call,
+* Python functions (if configuring `freezeyt` from Python, e.g. as a dict,
+  rather than YAML).
+
+
+An URL finder gets arguments:
+* filepath as `PurePath`
```

Comment:
A string (the path's `name`) would be better for a public API.

Reply:
Totally true I've updated